### PR TITLE
Make intersphinx links blue

### LINKED ISSUE #464: Having different link color for internal and external object references
Links are good, but we are reaching a point where we can see a lot of red in some pages.

Maybe all links to `@property` don't need to be in red... 

I suggest that we use the red color for links that points to an internal object, otherwise use a more regular color like black or blue.

### PR DESCRIPTION
**Pull request by tristanlatr**
Make intersphinx links blue. 

We now have a clear indication on whether the linked object is in the project or not.

Example parameters values and annotations: https://pydoctor--468.org.readthedocs.build/en/468/api/pydoctor.epydoc.markup.html#get_parser_by_name

Example base classes: https://pydoctor--468.org.readthedocs.build/en/468/api/pydoctor.test.test_sphinx.ClosingBytesIO.html

Fixes #464

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -72,6 +72,7 @@ What's New?
 
 in development
 ^^^^^^^^^^^^^^
+* Make intersphinx links blue instead of red. Keep red color for internal links only.
```
Comment (adiroiban) [inline README.rst:75]: I think that the important change is that you can now, as a theme designer, you can differentiate between internal and external documentation links. 

```suggestion
* Intersphinx links have now dedicated markup.
  With the default theme,
  this allows to have the external intershinx links blue while the internal links are red.
```

File: pydoctor/themes/base/apidocs.css [hunk 2]
```diff
@@ -371,6 +371,12 @@ code > a, #childList > div .functionHeader a {
     color:#c7254e;
     background-color:#f9f2f4;
 }
+
+/* Intersphinx links are not red, but simply blue */
+a.intersphinx-link {
+    color:#306998!important;
```
Comment (adiroiban) [inline pydoctor/themes/base/apidocs.css:377]: very minor suggestion

at least on my computer, with the grey background, this blue has little contrast and saturation. and looks like dark grey

```suggestion
    color:##06f!important;
```

File: README.rst (hunk 1 above)
Comment (adiroiban) [inline README.rst:75]: I think that the important change is that you can now, as a theme designer, you can differentiate between internal and external documentation links. 

```suggestion
* Intersphinx links have now dedicated markup.
  With the default theme,
  this allows to have the external intershinx links blue while the internal links are red.
```

**Review by adiroiban — APPROVED** [review]
Looks good. Thanks.

Only minor comments.

I think that it would help to also have markup for the internal apidoc links.

In this way you can have 3 types of links

* internal apidoc links
* external intersphinx links
* generic external links to non-documentation pages

**Comment by tristanlatr** [comment]
> I think that it would help to also have markup for the internal apidoc links. In this way you can have 3 types of links

We already have three types of css for links, but it’s implicit. The rules applied are the following: links are blue normal by default, links inside code tags are red, except if it’s an intersphinx link. I agree having another special css class for internal links would be good. I’ll try to add that to this PR.

**Comment by tristanlatr** [comment]
Please tell me if the contrast is Ok. 
I did not went with your color choice, i found it just too much intense blue.

**Review by adiroiban — APPROVED** [review]
Thanks. All good.

the color and contrast is a matter of taste and monitor capabilities :)

This can be merged. Thanks again for the update!

File: pydoctor/test/test_epydoc2stan.py [hunk 3]
```diff
@@ -390,7 +390,7 @@ def f():
         """
     ''', modname='test')
     html = docstring2html(mod.contents['f']).split('\n')
-    assert '<a href="test.SpanishInquisition.html" title="test.SpanishInquisition">SpanishInquisition</a>' in html
+    assert '<a href="test.SpanishInquisition.html" title="test.SpanishInquisition" class="internal-link">SpanishInquisition</a>' in html
```
Comment (tristanlatr) [inline pydoctor/test/test_epydoc2stan.py:393]:
```suggestion
    assert '<a href="test.SpanishInquisition.html" class="internal-link" title="test.SpanishInquisition">SpanishInquisition</a>' in html
```

**Pull request merged**